src/components/Terminal/index.js

```diff
@@ -57,15 +57,18 @@ const Terminal = ({ lines, title, noScroll }) => {
           <div className="code-wrapper">
             {lines && lines.map((line, index) => (
               <Fragment key={index}>
-                <pre className={`${line.short ? "short" : ""} ${line.color ? line.color : "blue"}`} >
-                  {line.indent &&
+                {line.code === "\n"
+                  ? <pre>{line.code}</pre>
+                  : <pre className={`${line.short ? "short" : ""} ${line.color ? line.color : "blue"}`} >
+                    {line.indent &&
```

Comment:
Why this change?

Reply:
Hi @Nikhil-Ladha,

Thanks for highlighting this. I forgot to include this console dev warning in the `Notes` section. It was occurring in this component `src/components/Learn-Components/what-await-section/index.js`.

>Warning: Expected server HTML to contain a matching text node for "
" in <pre>.
![image](https://github.com/layer5io/layer5/assets/90356410/33685060-1805-4c02-b66a-a71645279223)

When I re-examined this dev warning, I realized that while my changes did clear the dev warning, it was not a correct method. After looking into the issue more, I saw how the `"\n"` was handled in this component `src/sections/Meshery/Meshery-terminal/index.js`, by using a `" "` instead.

I reverted my changes on `src/components/Terminal/index.js` and then made the change to `" "` on `src/components/Learn-Components/what-await-section/index.js` and it cleared the warning. 

I also made changes to another similar usage of "\n" in this component `src/sections/Meshery/Meshery-features/index.js` (even though it was not triggering an error for some reason).

Thanks Nikhil!